### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by josemigallas**
See: https://github.com/aerogear/mobile-docs/pull/33

### CODE REVIEW

File: modules/ROOT/pages/push/sending-push-notifications.adoc [hunk 1]
```diff
@@ -13,10 +12,10 @@ The {push-ui} allows you to send a Push Notification to devices.
 
 . Open a PushApplication and click on the "Send Notification" button. The _Send Push_ dialog is displayed.
 
-. In the +Send Push+ dialog, enter text in the +Message+ text field. 
+. In the dialog, enter text in the +Message+ text field.
 
 . Click the +Send Push Notification+ button.
 
-NOTE: It is possible to filter the list of receivers, using +Alias+, +Device Types+ and +Category+. 
+NOTE: It is possible to filter the list of receivers by selecting specific variants using the +Variants+ drop-down and adding values to +Alias+, +Device Types+ and +Category+.
```
Comment (pb82) [inline modules/ROOT/pages/push/sending-push-notifications.adoc]: Should we explain how `alias` and `category` work?

**Comment by pb82** [comment]
@josemigallas something strange at the top of this file: `Unresolved directive in <stdin> - include::{partialsdir}/attributes.adoc[]`

**Comment by pb82** [comment]
@josemigallas under `Prerequiestes: Access to the {push-ui}.` it would be nice to add a small paragraph explaining the permissions you need to grant Openshift OAuth when you open the UI the first time. (You can reproduce the permissions dialog by opening the admin ui in a private tab)

File: modules/ROOT/pages/_partials/attributes.adoc [hunk 2]
```diff
@@ -5,32 +5,26 @@
...
 :keycloak-service: Identity Management
 :keycloak-ui: Keycloak admin UI
 :keycloak-dashboard: Auth Dashboard
     
-:unifiedpush-service: Push
 :push-ui: Unified Push admin UI
-   
-:digger-service: CI\CD
+:push-notification: push notification
+:push-notifications: push notifications
```
Comment (josemigallas) [inline modules/ROOT/pages/_partials/attributes.adoc]: @pb82 @finp 
Is this correct? Having constants like this? (singular/plural)

Comment (pb82) [reply modules/ROOT/pages/_partials/attributes.adoc]: No, i don't think this works unless both are defined. Do this instead: `:push-notifications:s`

File: modules/ROOT/pages/push/sending-push-notifications.adoc [hunk 3]
```diff
@@ -2,20 +2,29 @@ include::{partialsdir}/attributes.adoc[]
...
 
-NOTE: It is possible to filter the list of receivers, using +Alias+, +Device Types+ and +Category+. 
+. Click on *Send Push Notification* button.
 
-Your message is delivered by the 3rd party Push Network to your device.
+Your message is now delivered by the 3rd party Push Network to your device.
+
+TIP: You can give a higher priority to a push message by setting its priority to 'high' instead of 'normal'.
+
+== Filtering recipients
```
[the remark this answers is not in the document]
Comment (pb82) [reply modules/ROOT/pages/push/sending-push-notifications.adoc]: Ok, but we should at least put in a sentence and a link to the section in `Registering Device` where the details are explained.

**Comment by josemigallas** [comment]
@pb82 
Regarding the missing _permissions_ you proposed, I think the place for that is in Push > Provisioning. Grating those permissions is a setup step to fully configure UPS in my opinion and it doesn't relate with the procedure of sending a push. WDYT?

**Comment by pb82** [comment]
@josemigallas i think those permissions are best described in the admin ui document that @jhellar is working on.

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+. Click *Send Push Notification* to send the message to the target application.
+
+NOTE: The 3rd party Push Network is responsible for delivering the Push Notification to the target application.
+
+[discrete]
+== Additional Resources
+
+=== Filtering recipients
+
+It is possible to select a more specific set of recipients by adding additional values to *Variants*, *Alias*, *Device Types* and *Category*.
```
Comment (jstaffor) [inline modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: This part defo requires a review Jose

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+The {push-ui} allows you to send {push-notification}s to devices.
+
+[discrete]
+== Prerequisites
+
+* Access to the {push-ui}.
+* An application with at least one binding or variant.
+
+[discrete]
+== Procedure
```
Comment (jstaffor) [inline modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: I've stripped this down, do review and make sure I haven't lost anything important

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+NOTE: The 3rd party Push Network is responsible for delivering the Push Notification to the target application.
+
+[discrete]
+== Additional Resources
+
+=== Filtering recipients
+
+It is possible to select a more specific set of recipients by adding additional values to *Variants*, *Alias*, *Device Types* and *Category*.
+A {mobile-client} must be associated with one or more of these parameters in order for the filtering to work.
+Filtering is done when you xref:ROOT:push/coding.adoc[Register a Device].
```
Comment (josemigallas) [inline modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: @jstaffor What is done during the registration is that those parameters (alias, category, etc) are set.

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+[discrete]
+== Procedure
+
+. Open the {push-ui} in a browser.
+
+. Select the target application from the home page and click *Send Notification To This App*.
+
+. Enter text in the *Message* form.
++
+NOTE: The default priority of a Push Notification is `normal`. To increase the priority, change *Priority* to `high`.
```
Comment (josemigallas) [inline modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: Now that you added "additional resources" I think we could move this note to its own paragraph.

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc (hunk 6 above)
Comment (jstaffor) [reply modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: Ok, I'll make the update. Re the heading, "=== Filtering recipients" - can you think of a more accurate/descriptive heading?

File: modules/ROOT/pages/push/proc_sending-a-push-notification.adoc (hunk 7 above)
Comment (jstaffor) [reply modules/ROOT/pages/push/proc_sending-a-push-notification.adoc]: I understand the thinking here, however, I think the reader needs to read this as they move through the procedure as we are not guaranteed that the reader will read the reference material.

**Pull request merged**